{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "ad53859a_b05ed6f7",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 13252
      },
      "writtenOn": "2022-05-24T20:16:52Z",
      "side": 1,
      "message": "recheck",
      "revId": "7f14e760b0dbbb61205ac357080964521e26754e",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "397762c0_dbc5d92c",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 13252
      },
      "writtenOn": "2022-05-25T06:40:26Z",
      "side": 1,
      "message": "will need to wait until tempest py36 issues are fixed",
      "revId": "7f14e760b0dbbb61205ac357080964521e26754e",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}